docs/website/mkdocs.yml

```diff
@@ -94,11 +94,11 @@ markdown_extensions:
 # Note: may include external links and titles are optional for internal links
 nav:
   - Home: 'index.md'
-  - 'ML frameworks':
-      - 'ml-frameworks/index.md'
-      - TensorFlow: 'ml-frameworks/tensorflow.md'
-      - TensorFlow Lite: 'ml-frameworks/tensorflow-lite.md'
```

Comment:
This is linked to at the bottom of https://github.com/google/iree/blob/main/docs/website/docs/blog/2021-07-19-tflite-tosa.md#examples. Can you update that ref?

Reply:
Done.